projects/bp-gallery/src/components/views/geoguessr/GeoView.tsx

```diff
@@ -63,18 +68,29 @@ const GeoView = () => {
   const archives: FlatArchiveTag[] | undefined =
     useSimplifiedQueryResponseData(picturesData)?.archiveTags;
 
+  const isGeoCollectionPictures = useFlag('geopictures_from_collection');
+  const geoCollectionId = '357';
```

Comment:
Maybe put the `'357'` as a feature value, so we can change it via growthbook?

Reply:
I already created a collection on main that's called "GeoSpiel Bilder" with this ID, but growthbook may be a good idea